src/bin/nps.js

```diff
@@ -82,6 +82,10 @@ function getPSConfig() {
   return config
 }
 
+function getPSConfigFilepath() {
+  return program.config || findUp.sync('package-scripts.yml') || findUp.sync('package-scripts.js')
```

Comment:
Hmmm... I'm thinking that more people will use the `.js` than those who use the `.yml`. Could you put the `.js` first here?